cypress/integration/hosts/hosts_sorting.ts

```diff
@@ -15,7 +15,8 @@ const sortByTests = [
       "build10.ny.cbi.10gen.cc",
     ],
   },
-  {
+  /* TODO EVG-18907: Re-enable distro sort upon completion of EVG-18900
```

Comment:
https://github.com/evergreen-ci/evergreen/pull/6255#pullrequestreview-1293333020

@ybrill  fixed this you should be able to re enable the tests @sophstad